doc/manual/import.rst

```diff
@@ -1,17 +1,56 @@
-Adding contest data programmatically
-====================================
+Adding contest data in bulk
+===========================
 
-DOMjudge offers API endpoints to add or update contest data programmatically.
+DOMjudge offers two ways to add or update contest data in bulk: using API
+endpoints and using the jury interface.
 In general, we follow the `CCS specification`_ for all file formats involved.
 
-All of the following examples require you to set up admin credentials in your
-`.netrc`_ file. You need to install `httpie`_ and replace the
+For using the API, the following examples require you to set up admin credentials
+in your `.netrc`_ file. You need to install `httpie`_ and replace the
 ``<API_URL>`` in the examples below with the API URL of your local DOMjudge
 installation.
 
 Importing team categories
 -------------------------
 
+There are two formats to import team categories: a JSON format and a legacy TSV format.
+
+Using JSON
+^^^^^^^^^^
+
+Prepare a file called ``groups.json`` which contains the team categories.
+It should be a JSON array with objects, each object should contain the following
+fields:
+
+- ``id``: the category ID
```

Comment:
Something you can just make up?

Reply:
Good point. I think they should at least be integer so I will add something